.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2
+
+jobs:
+
+  build-python27: &job-template
```

Comment:
Circle CI switched from 1.0 to 2.0 in August last year which is very different (`.circle/config.yml` instead of `.circleci.yml`, job/workflow based semantics, containers based directly on docker etc.)